q2_gneiss/plot/_plot.py

```diff
@@ -173,7 +178,8 @@ def f(x):
 
             metadata = pd.DataFrame({dcat.name: dcat})
             top_num_features = num_fold_change.index[-n_features:]
-            top_denom_features = num_fold_change.index[:n_features]
+            top_denom_features = denom_fold_change.index[:n_features]
```

Comment:
I think this is still taking the bottom of the sort, should it be `[-n_features:]`?